index.js

```diff
@@ -362,15 +374,15 @@ module.exports = class Filter extends Plugin {
 
   isDirectory(relativePath, entry) {
     // @ts-ignore
-    if (this.inputPaths === undefined) {
+    if (this.fsMerger === undefined) {
```

Comment:
when is this `undefined` ?

Reply:
If this.isDirectory is called before `build` method is called.

Reply:
Removed it